sru-scripts/README.md

```diff
@@ -0,0 +1,17 @@
+Ubuntu SRU xenial, bionic, cosmic, disco (post-freeze): %SRU_START_DATE%
```

Comment:
I'm not 100% sure what this file is doing here. I would expect that we would no longer be copying the scripts (because they don't need copy-time substitution performed on them as the templates did).

Reply:
That file a sourced README template that sru-create.py uses to generate the top-level SRU verification table at <SRU_DATE>/README.md.   I've renamed it SRU_README.tmpl and updated sru-create.py to source it properly.